scipy/stats/kde.py

```diff
@@ -447,6 +447,12 @@ def resample(self, size=None):
             The number of samples to draw.  If not provided, then the size is
             the same as the effective number of samples in the underlying
             dataset.
+        seed : None or int or ``numpy.random.RandomState`` instance, optional
```

Comment:
Could you change the docstring to:
```
    seed : int or `np.random.RandomState`, optional
        If `seed` is not specified the `np.RandomState` singleton is used.
        If `seed` is an int, a new `np.random.RandomState` instance is used,
        seeded with seed.
        If `seed` is already a `np.random.RandomState instance`, then that
        `np.random.RandomState` instance is used.
        Specify `seed` for reproducible drawing of random variates.
```

Reply:
Should `np.RandomState` in the second line be `np.random.RandomState`?

Reply:
Perhaps change to:
```
If `seed` is None, return the RandomState singleton used by np.random.
```

Reply:
I think that this function does not return any `RandomState`. Thus, I have changed to:
``If `seed` is None, random variates are drawn by the RandomState singleton used by np.random.``
Let me know if any further changes are required.